src/styles/blocks/header.scss

```diff
@@ -0,0 +1,69 @@
+header {
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  display: flex;
+  justify-content: space-between;
+
+  .logo {
+    display: inline;
+    float: left;
+    width: 40px;
+    height: 40px;
+    margin-top: 10px;
+  }
+}
+
+.nav {
+  font-size: 12px;
+  text-transform: uppercase;
+
+  &__list {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-items: center;
+    margin: 0;
+    padding: 0;
+    border: 0;
+  }
+
+  &__item {
+    display: block;
+    padding: 0 10px;
+
+    &:first-child {
+      padding-left: 0;
+    }
+
+    &:last-child {
+      padding-right: 0;
+    }
+
+    a {
```

Comment:
Don't style by tags! Check the 2nd point from the [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)

Reply:
a were changed to nav__link class